riscv/src/code_gen.rs

```diff
@@ -1211,22 +1222,25 @@ fn process_instruction<A: InstructionArgs>(instr: &str, args: A) -> Result<Vec<S
             let (r1, r2, label) = args.rrl()?;
             let label = escape_label(label.as_ref());
             vec![format!(
-                "branch_if_zero {}, {}, 0, {label};",
+                "branch_if_diff_equal {}, {}, 0, {label};",
                 r1.addr(),
                 r2.addr()
             )]
         }
         "beqz" => {
             let (r1, label) = args.rl()?;
             let label = escape_label(label.as_ref());
-            vec![format!("branch_if_zero {}, 0, 0, {label};", r1.addr())]
+            vec![format!(
+                "branch_if_diff_equal {}, 0, 0, {label};",
+                r1.addr()
+            )]
         }
         "bgeu" => {
             let (r1, r2, label) = args.rrl()?;
             let label = escape_label(label.as_ref());
-            // TODO does this fulfill the input requirements for branch_if_positive?
+            // TODO does this fulfill the input requirements for branch_if_diff_greater_than?
```

Comment:
What does that even mean?

Reply:
no idea, leftover from the very first implementation from @chriseth